crew

```diff
@@ -739,27 +740,20 @@ def resolve_dependencies
 
   return if @dependencies.empty?
 
+  puts "\nThe following packages also need to be installed:\n\n"
 
-  puts 'The following packages also need to be installed: '
-
-  i = 0
-  last_deps = []
-  last_packages = ["curl", "ghc", "mandb", "gtk3", "sommelier"]
+  last_packages = ['ghc', 'gtk3', 'sommelier']
+  
   @dependencies.each do |dep|
-    if last_packages.include?(dep)
-      @dependencies.delete_at(i)
-      last_deps.push(dep)
+    if last_packages.include?(dep) then
+      @dependencies = (@dependencies - [dep] + [dep])
```

Comment:
I think it's also important to install gtk3 before sommelier.  Does this logic ensure that will happen?

Reply:
No, but `sommelier` is the last dep in a package at most time, so it may not a big problem :)